### DISCUSSION TITLE
Multiple Seedboxes

### DISCUSSION THREAD

**Discussion by Friday13th87** [autobrr/autobrr#1303] [Q&A]
Is there a way to use autobrr with multiple seedboxes, that autobrr is sending the torrents round by round to the seedboxes, so that the seedboxes do not get the same torrents?
so for example
1 --> SB 1, 2 --> SB 2, 3 --> SB 1, 4 --> SB 2 ....

**Comment by zze0s** [comment]
Hi. Currently that's not supported with anything built-in. It's been on the roadmap since the start tho to have a "smart scheduler". I did start building a separate one for qBittorrent but it needs more polish.

What client(s) are you using?

**Comment by Friday13th87** [comment]
i am using qbittorrent for now
